website_multi_theme/README.rst

```diff
@@ -0,0 +1,149 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Website Multi Theme
+===================
+
+Allow the website admin to set a different theme for each website.
+
+Installation
+============
+
+To make this module work, you need to either:
+
+* Install any of the officially supported themes:
+
+  * theme_bootswatch
+
+* Install any of the unofficially supported themes (at your own risk):
+
+  * theme_anelusia
+  * theme_artists
+  * theme_avantgarde
+  * theme_beauty
+  * theme_bewise
+  * theme_bistro
+  * theme_bookstore
+  * theme_clean
+  * theme_enark
+  * theme_graphene
+  * theme_kea
+  * theme_loftspace
+  * theme_mongolia
+  * theme_nano
+  * theme_notes
+  * theme_odoo_experts
+  * theme_orchid
+  * theme_treehouse
+  * theme_vehicle
+  * theme_yes
+  * theme_zap
+
+Themes in the above lists will become multiwebsite when installed along this
+module. **If they get installed after ``website_multi_theme``, update this
+module manually**, or it will not be notified of such change.
```

Comment:
Spitballing here, but what about an override in [`ir.module.module.upstream_dependencies`](https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/module/module.py#L455) & inject this module in the event that the category is `Theme`? From what I understand, injecting it into the dependencies would cause it to get updated right?

Odoo already seems to use this category for other logic such as the theme installer, so I think it would be a safe target here too.